test/cases/testdata/strings/test-strings-0924.yaml

```diff
@@ -0,0 +1,12 @@
+cases:
+- note: 'strings/repeat_test'
+  modules:
+  - |
+    package generated
```

Comment:
Nit: `generated` -> `test`. Also it would be helpful to add a test case with bad operands.

Reply:
updated and added a test